src/main/java/org/primeframework/mvc/sampleapp/session/OurUser.java

```diff
@@ -0,0 +1,7 @@
+package org.primeframework.mvc.sampleapp.session;
```

Comment:
Should we use the `domain` package for all domain objects for the sample?

Reply:
Done